### DISCUSSION TITLE
Not working with AuthMe

### DISCUSSION THREAD

**Discussion by CrafterOscar** [games647/FastLogin#711] [Q&A] [answered]
When I use the plugin with AuthMe on a bungeecord network, sometimes when I switch servers it asks me to login again, even tho I have a premium account and I executed the /premium command. When I type the password it says I'm already logged in, which isn't true because the message keeps getting sent. I have everything configured properly and both plugins are up to date on both bungeecord and spigot servers. It only happens like 50% of the time when I switch servers, otherwise it works as it's suposed to. Could it be a problem with the database, which could be slow? I have tried everything and I have no idea how to fix it. Any help is greatly appreciated!

**Comment by TuxCoding** [comment, marked as answer]
This happens, because sometimes AuthMe is faster with the processing than FastLogins default delay. This requires some modifications on AuthMe end.

**Reply by CrafterOscar** [reply]
Ok, thanks for your help
